actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -581,15 +581,24 @@ def current_page?(options = nil, check_parameters: false, **options_as_kwargs)
       # If +name+ is not specified, +phone_number+ will be used as the name of
       # the link.
       #
+      # A +country_code+ option is supported, which prepends a plus sign and the
+      # given country code to the linked phone number. For example,
+      # <tt>country_code: "01"</tt> will prepend <tt>+01</tt> to the linked
+      # phone number.
+      #
       # Additional HTML attributes for the link can be passed via +html_options+.
       #
       # ==== Options
+      # * <tt>:country_code</tt> - Prepends the country code to the phone number
```

Comment:
Nitpicking, but this line is "prepends" (cf. "preset" and not "presets" for `:body` below) and does not end in a "." (but the one for `:body` does) 🐞👌